app/templates/offers/offer_details.hbs

```diff
@@ -86,12 +32,105 @@
+
+            <div class='time_indicator'>
+              {{!-- Submitted Offer --}}
+              {{#if offer.isSubmitted}}
+                Submitted <i class="fa fa-clock-o"></i>
+                {{view 'time-ago' timeValue=offer.submittedAt smallPrefix=true}}
+
+              {{!-- Offer Under Review --}}
+              {{else if offer.isUnderReview}}
+                {{t "review_offer.review_started_by" firstName=offer.reviewedBy.firstName lastName=offer.reviewedBy.lastName}}
+                <i class="fa fa-clock-o"></i>
+                {{view 'time-ago' timeValue=offer.reviewedAt smallPrefix=true}}
 
-      <div class="small-1 columns">
-        {{#if offer.unreadMessagesCount}}
-          <span class='unread_length'>{{offer.unreadMessagesCount}}</span>
-        {{/if}}
+              {{!-- Offer Completed with Review --}}
+              {{else if offer.isReviewed}}
+                {{t "review_offer.reviewed"}}
+                <i class="fa fa-clock-o"></i>
+                {{view 'time-ago' timeValue=offer.reviewCompletedAt smallPrefix=true}}.
+                {{t "review_offer.plan_transport"}}
+
+              {{!-- Closed Offer --}}
+              {{else if offer.isClosed}}
+                Closed by {{offer.createdBy.firstName}} {{offer.createdBy.lastName}}
+                <i class="fa fa-clock-o"></i>
+                {{view 'time-ago' timeValue=offer.reviewCompletedAt smallPrefix=true}}
+
+              {{!-- Received Offer --}}
+              {{else if offer.isReceived}}
+                Received by {{offer.createdBy.firstName}} {{offer.createdBy.lastName}}
+                <i class="fa fa-clock-o"></i>
+                {{view 'time-ago' timeValue=offer.receivedAt smallPrefix=true}}
+
+              {{!-- Scheduled Offer --}}
+              {{else if offer.isScheduled}}
+
+                {{#if offer.delivery.isAlternate}}
+                  {{t 'offer.offer_details.is_collection'}}
+                {{else if offer.delivery.isDropOff}}
+                  {{t 'offer.offer_details.is_drop_off'}}
+                {{else if offer.delivery.isGogovan}}
+                  {{#if offer.delivery.gogovanOrder.isActive}}
+                    {{t 'offer.offer_details.is_gogovan_confirm'}}
+                   {{else}}
+                    {{t 'offer.offer_details.is_gogovan_order'}}
+                  {{/if}}
+                {{/if}}
+
+                <i class="fa fa-truck"></i>
+                <b>
+                  {{#if offer.delivery.isGogovan}}
+                    {{offer.delivery.schedule.slotName}},
+                  {{/if}}
+
+                  {{schedule-time-status offer.delivery.schedule.scheduledAt}}
+
+                  {{#if offer.delivery.noGogovan}}
+                    , {{offer.delivery.schedule.dayTime}}
+                  {{/if}}
+                </b>
+
+              {{/if}}
+            </div>
+
+            <div>
+              {{#if offer.isReceived}}
+                {{offer.approvedItems.length}} Received,
```

Comment:
@steveyken We will add it now. These translations are missed out earlier. Thanks.